### DISCUSSION TITLE
Looking for partnership

### DISCUSSION THREAD

**Discussion by ivopc** [devshareacademy/project-roadmap#47] [General]
Hello, I am a full stack web and game developer looking for partnership. You can check my: website, GitHub and LinkedIn in my GitHub profile.
At the moment I've a Pokémon based MMORPG called Monster Valle that I made with Phaser and NodeJS and some other projects, and worked to gamedev companies too, I want to turn it into a (MMO)RPG Game Engine like RPG Maker. I look that you are doing a developer academy and that is very interesting to me too. So are you interested in a partnership?

**Comment by scottwestover** [comment]
Hi @ivopc , thanks for reaching out. I would be interested in discussing this further, and to get your thoughts around what you had in mind.  I will follow up with you via email, so we can discuss further offline.
